jsonrpc/src/jsonrpc.mli

```diff
@@ -93,6 +93,7 @@ module Response : sig
         | ServerCancelled
         | ContentModified
         | RequestCancelled
+        | Custom of int
```

Comment:
Could you rename this to `Other`? `Custom` is a fine name, but we use `Other` elsewhere in lsp

Reply:
Sure, done!